Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -525,4 +541,76 @@ namespace Multiplayer
         constexpr bool autoRequeue = true;
         m_connectionEvent.Enqueue(AZ::SecondsToTimeMs(retrySeconds), autoRequeue);
     }
+
+    void MultiplayerEditorSystemComponent::PopulateEditorGlobalContextMenu(QMenu* menu, const AZ::Vector2& point, [[ maybe_unused ]] int flags)
+    {
+        menu->setToolTipsVisible(true);
+
+        m_contextMenuViewPoint = point;
+
+        QAction* action = nullptr;
+
+        action = menu->addAction(QObject::tr("Create multiplayer entity"));
+        action->setShortcut(QKeySequence(Qt::CTRL + Qt::ALT + Qt::Key_M));
+        QObject::connect(
+            action, &QAction::triggered, action,
+            [this]
+        {
+            ContextMenu_NewMultiplayerEntity();
+        });
```

Comment:
While it should not cause crashes or issues (since there's more checks down the line), you should also check whether the parent entity is read-only before displaying the menu item. Else the menu item will display but do nothing if clicked from inside a procedural prefab.

```
    readOnlyEntityPublicInterface = AZ::Interface<AzToolsFramework::ReadOnlyEntityPublicInterface>::Get();
    if(readOnlyEntityPublicInterface && readOnlyEntityPublicInterface->IsReadOnly(selectedEntityId))
    {
        // ...
    }
```

(This will be streamlined soon)